src/server/req_runjob.c

```diff
@@ -985,11 +985,15 @@ svr_strtjob2(job *pjob, struct batch_request *preq)
 
 	/* send the job to MOM */
 
+	/* clearing old comment since we are sending a fresh job */
+	job_attr_def[(int) JOB_ATR_Comment].at_decode(
+			&pjob->ji_wattr[(int) JOB_ATR_Comment], (char *) 0,
+			(char *) 0, "");
```

Comment:
I have posted a EDD ( https://pbspro.atlassian.net/wiki/display/PD/EDD+for+PP-374%3A+Job+with+a+root-owned+script+runs+even+if+root_reject_scripts+set+to+true# ) for the proposed changes to update attribute comment with "Job has sent for execution at $execvnode" just before sending job for execution. Please review and provide feedback.